src/GeneralStateTestsFiller/stPreCompiledContracts/idPrecompsFiller.yml

```diff
@@ -397,9 +405,12 @@ idPrecomps:
       gas: !!int -1
       value: !!int -1
     network: 
-    - ">=Istanbul"
```

Comment:
must remain >=

Reply:
I did that for the `:label yes` and `:label no`.

I added a third label, `:label from_cancun`, for the precompile that cancun adds.